osbs-client.spec

```diff
@@ -15,7 +15,7 @@
 %global with_check 1
 %endif
 
-%global commit 5fa7fa287df56468146f13285c4ea42f557e154a
+%global commit cfcf69702c0f8e849c6d3624d119dbf0d4443498
```

Comment:
Is this intentional? I think we generally only change this when cutting a new release.

Reply:
It was intentional. After the release was cut, version was bumped to 0.54-0.dev.git.%{shorcommit}, which led me to believe this would not be an issue.

Without pointing the spec file to the new reference, rpmbuild will throw warnings about duplicated entries in %files (although the customization files will not get packaged).